src/SWP/Bundle/WebRendererBundle/Theme/Configuration/TenantableConfigurationProvider.php

```diff
@@ -63,14 +63,19 @@ public function __construct(
      */
     public function getConfigurations()
     {
-        $configs = array_map(
-            [$this->loader, 'load'],
-            $this->fileLocator->locateFilesNamed($this->configurationFilename)
-        );
+        // Handle there being no theme.json file
+        try {
```

Comment:
I just realized that when throwing exception here it, we will see white page in production env. Silencing it here is the only option I see for now. But the listener should either notify that there are no themes added at all or that user should install default theme. Themes are loaded on kernel boot so to display a message it should pass this code and exception should be thrown later as it is now in this PR